Tensile/LibraryLogic.py

```diff
@@ -102,49 +112,38 @@ def analyzeProblemType( problemType, problemSizeGroups, inputParameters ):
       line += "\n"
     print(line)
 
-  ######################################
-  # Print solutions used
-  print1("# Solutions Used:")
-  for i in range(0, len(logicAnalyzer.solutions)):
-    s = logicAnalyzer.solutions[i]
-    s["SolutionIndex"] = i
-    s["SolutionNameMin"] = Solution.getNameMin(s, solutionMinNaming)
-    print1("(%2u) %s : %s" % (i, \
-        Solution.getNameMin(s, solutionMinNaming), \
-        Solution.getNameFull(s)))  # this is the right name
-
-  selectionSolutionsIdsList = None 
-  selectionSolutions = None
   if enableTileSelection:
-    validSelectionSolutions = SolutionSelectionLibrary.analyzeSolutionSelection(problemType, problemSizeGroups)
+    if globalParameters["NewClient"] == 2:
+      validSelectionSolutions = SolutionSelectionLibrary.analyzeSolutionSelection(problemType, selectionFileNameList, \
```

Comment:
Why do the old and new clients required different analysis?

Reply:
The tile aware is a different model and hence different analysis/logic/algorithm.

Reply:
But this is not tile aware vs. not, it's old client vs. new.

Reply:
But it is tile aware vrs. not tile aware. 

The old version of the tile aware, which was created a year ago, generated a different output format.  This implementation used the format generated from your client app. 

Eventually when the old client gets phased out this will reduce to one case.